src/main/java/com/laytonsmith/core/events/drivers/PlayerEvents.java

```diff
@@ -1537,9 +1540,17 @@ public boolean modifyEvent(String key, Construct value, BindableEvent event) {
                     }
                 }
 				if("format".equals(key)){
+					String format = value.nval();
+					if(format == null) {
+						throw new CRENullPointerException("The \"format\" key in " + modify_event.class.getSimpleName() + " for the " + this.getName()
+								+ " event may not be null.", Target.UNKNOWN);
+					} else if(format.replaceAll("%%", "").replaceAll("\\%\\%|\\%[12]\\$s", "").contains("%")) {
```

Comment:
Seems inefficient to run this on every chat event. This can be combined into one precompiled pattern. Maybe it should be a compile time check?